account_credit_control/data/data.xml

```diff
@@ -4,21 +4,17 @@
     <record id="email_template_credit_control_base" model="mail.template">
         <field name="name">Credit Control Email</field>
         <field name="email_from">${user.company_id.email or ''}</field>
-        <field name="subject">Credit Control:
-            (${object.current_policy_level.name or 'n/a'})
-        </field>
+        <field name="subject">Credit Control: (${object.policy_level_id.name or 'n/a'})</field>
         <field name="email_to">${object.get_email() or  ''}</field>
         <field name="model_id" ref="model_credit_control_communication"/>
-        <field name="auto_delete" eval="True"/>
-        <field name="lang">
-            ${object.get_contact_address().lang or 'en_US'}
-        </field>
+        <field name="auto_delete" eval="False"/>
+        <field name="lang">${object.contact_address_id.lang or 'en_US'}</field>
         <field name="report_template" ref="credit_control_summary"/>
         <field name="body_html"><![CDATA[
-  Dear ${object.contact_address.name or ''}
+  Dear ${object.contact_address_id.name or ''}
   <br/>
   <br/>
-  ${object.current_policy_level.custom_mail_text | safe}
+  ${object.policy_level_id.custom_mail_text | safe}
```

Comment:
This change breaks no updated templates as well

Reply:
Hmm same thing.. did you see the migration script